doc/ref_model/chapters/chapter06.rst

```diff
@@ -60,178 +107,223 @@ The Or-Vi and Vi-Vnfm are both specifying interfaces provided by the VIM and the
++-----------------------+------------------------+-------+--------+---------------+------------------------------------+
+| Request               | Response               | From, | Type   | Parameter     | Description                        |
+|                       |                        | To    |        |               |                                    |
++=======================+========================+=======+========+===============+====================================+                       |                        |       | Input  | accFilter     | the accelartor sub-system(s) to    |                       |                        |       |        |               | initialize and retrieve their      |                       |                        |       |        |               | capabilities.                      |                       |                        |       +--------+---------------+------------------------------------+ InitAccRequest        | InitAccResponse        | VNF → | Filter | accAttributeS | attribute names of accelerator     |                       |                        | NFVI  |        | elector       | capabilities                       |                       |                        |       +--------+---------------+------------------------------------+                       |                        |       | Output | accCapabilitie| acceleration sub-system            |                       |                        |       |        | s             | capabilities                       |-----------------------+------------------------+-------+--------+---------------+------------------------------------+                       |                        |       | Input  | accEvent      | event the VNF is interested in     |                       |                        |       +--------+---------------+------------------------------------+ RegisterForAccEventRe | RegisterForAccEventRes | VNF → | Input  | vnfEventHandl | the handler for NFVI to use when   | quest                 | ponse                  | NFVI  |        | erId          | notifying the VNF of the event     |-----------------------+------------------------+-------+--------+---------------+------------------------------------+                       |                        | NFVI  | Input  | vnfEventHandl | Handler used by VNF registering    |                       |                        | → VNF |        | erId          | for this event                     | AccEventNotificationR | AccEventNotificationRe |       +--------+---------------+------------------------------------+ equest                | sponse                 |       | Input  | accEventMetaD |                                    |                       |                        |       |        | ata           |                                    |-----------------------+------------------------+-------+--------+---------------+------------------------------------+
```

Comment:
Something screwed up here....

Reply:
Good catch. Fixed in [1112060](https://github.com/cntt-n/CNTT/pull/2931/commits/11120609e6cda9a655d10194dcbacb9987210281)